app/models/leader.rb

```diff
@@ -62,6 +62,13 @@ def slack_update
     end
   end
 
+  def timezone
+    Timezone.lookup(latitude, longitude)
+  rescue Timezone::Error::InvalidZone
+    Rails.logger.warn("Unable to find timezone for leader \##{id}")
```

Comment:
Not sure this log statement is inline with the other precedent in the code base.